data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReport.kt

```diff
@@ -10,8 +10,10 @@ data class GaeFeedbackReport(
 
   /** The current version of the feedback report used. */
   @Json(name = "android_report_info_schema_version") val schemaVersion: Int,
-  /** The date and time in ms that the report was created. */
+  /** The date and time in sec that the report was submitted. */
   @Json(name = "report_submission_timestamp_sec") val reportSubmissionTimestampSec: Int,
```

Comment:
Should this be `Long` instead of `Int`?

Reply:
This is defined as an int since the backend timestamp for seconds uses an int rather than a double so I kept the consistency. The long => int conversion could happen later, though, (i.e. this could remain a Long and then we perform the conversion in the backend controller) -- would that be preferred? I don't see if there's any benefit to keeping this a Long.